source/agora/common/crypto/ECC.d

```diff
@@ -168,6 +182,21 @@ public struct Scalar
     }
 }
 
+nothrow @nogc @safe unittest
+{
+    assert(Scalar(`0x1000000000000000000000000000000014def9dea2f79cd65812631a5cf5d3ec`).isValid);
+}
+
+nothrow @nogc @safe unittest
+{
+    assert(!Scalar(`0x1000000000000000000000000000000014def9dea2f79cd65812631a5cf5d3ed`).isValid);
+}
+
+nothrow @nogc @safe unittest
+{
+    assert(!Scalar(`0x0000000000000000000000000000000000000000000000000000000000000000`).isValid);
+}
```

Comment:
These should be merged into a single unittest.

Reply:
Done